### DISCUSSION TITLE
请问下这个bug需要怎么处理

### DISCUSSION THREAD

**Discussion by Mio521** [mikumifa/biliTickerBuy#340] [提问专区]
<img width="1280" alt="760976b3142564228612b177ba335626" src="https://github.com/mikumifa/biliTickerBuy/assets/174247197/551d8866-c942-4da3-add4-79c279bcc59a">
如图划线部分 还有下面那个文件需要下载吗？谢谢🙏🏻

**Comment by Mio521** [comment]
> <img alt="760976b3142564228612b177ba335626" width="1280" src="https://private-user-images.githubusercontent.com/174247197/344648708-551d8866-c942-4da3-add4-79c279bcc59a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4MzE2ODEsIm5iZiI6MTcxOTgzMTM4MSwicGF0aCI6Ii8xNzQyNDcxOTcvMzQ0NjQ4NzA4LTU1MWQ4ODY2LWM5NDItNGRhMy1hZGQ0LTc5YzI3OWJjYzU5YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwMVQxMDU2MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZmYwMGYzNDYzNDZlMDMxZTkwYWZjYjFkYjcxZTJjMmRkZTQyMGViMjBkYTA1NjNiOGFjN2MyMTY4MzMwZjI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.62IEBqAP-X8pMufNEFEKdhJQxFhs_OzgntJIMGRpxmY"> 如图划线部分 还有下面那个文件需要下载吗？谢谢🙏🏻
第一张图片上传错误 应为第二张
![1a9ff3713bf901447d6ab39391e9403c](https://github.com/mikumifa/biliTickerBuy/assets/174247197/dd89c050-f3bf-406e-a3e2-c074052ea1f9)

**Comment by mikumifa** [comment]
看一下wiki

**Comment by mikumifa** [comment]
https://github.com/mikumifa/biliTickerBuy/wiki
